File: code-samples/java/refunds.java
Map<String, Object> params = new LinkedHashMap<String, Object>();
            String uniqueRequestId = UUID.randomUUID().toString().substring(0, 12);
            params.put("amount", 10);
            params.put("unique_request_id", uniqueRequestId);
            params.put("order_id", this.order.getOrderId());
            
            RequestOptions requestOptions = RequestOptions.createDefault().withCustomerId(customerId);
            Refund refund = Order.refund(params, requestOptions);

File: code-samples/php/refunds.php
$params = array ();
        $params ['order_id'] = "order_id";
        $params['unique_request_id'] = uniqid('php_sdk_test_');
        $params['amount']= "order amount";
        $requestOption = new RequestOptions();
       
$requestOption->withCustomerId("testing-customer-one")->withMerchantId(“merchant_id”); 
        $order = Order::refund($params, $requestOption);
        echo "amout refunded: " . $order->amountRefunded . PHP_EOL;
